.github/workflows/packages-build-multiple-managers.yml

```diff
@@ -0,0 +1,95 @@
+name: Build multiple Wazuh Manager Packages - DEB|RPM - amd64|arm64
+on:
+  workflow_dispatch:
+    inputs:
+      build-amd64-deb:
+        description: 'Build .deb wazuh-manager on amd64 arch'
+        default: false
+        type: boolean
+        required: false
+      build-amd64-rpm:
```

Comment:
Should we change 'amd64' to 'x86_64' to be clearer for users, even though we continue processing it as 'amd64'? What do you think?